test/utils/mesh_test_utils.ts

```diff
@@ -87,13 +87,17 @@ export class MeshTestUtils {
         // NOTE(jalextowle): The way that Mesh validation currently works allows us
         // to only set the maker balance a single time. If this changes in the future,
         // this logic may need to be added to `addOrdersAsync`.
+        // tslint:disable-next-line:await-promise
```

Comment:
This isn't a promise?

Reply:
TSLint is being silly, it's technically a `PromiseWithTransactionHash` 😂